src/pkg/packager/dev.go

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package packager contains functions for interacting with, managing and deploying Zarf packages.
+package packager
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/defenseunicorns/zarf/src/config"
+	"github.com/defenseunicorns/zarf/src/internal/packager/validate"
+	"github.com/defenseunicorns/zarf/src/pkg/message"
+	"github.com/defenseunicorns/zarf/src/types"
+)
+
+// DevDeploy creates + deploys a package in one shot
+func (p *Packager) DevDeploy() error {
+	config.CommonOptions.Confirm = true
+	p.cfg.CreateOpts.SkipSBOM = !p.cfg.CreateOpts.IsYOLO
```

Comment:
Pretty sure this logic is backward - it will never do SBOMs really this way

Reply:
whoops, forgot `SkipSBOM` was also a negation boolean